src/actions/account/index.ts

```diff
@@ -40,34 +44,24 @@ export const checkIdentityExists: ThunkAction = async (
   getState,
   backendMiddleware,
 ) => {
-  const { keyChainLib, storageLib, encryptionLib } = backendMiddleware
-  const encryptedEntropy = await storageLib.get.encryptedSeed().catch(err => {
-    // TODO Fix this
-    if (err.message.indexOf('no such table') === 0) {
```

Comment:
With `synchronization: true` this could not have happen at all, same goes for the migration setup now... Would be interesting to know where this error came from.

Reply:
The Tables are always there when this fires (since that fires in app init). Not 100% relevant anymore.